components/Ink/src/private/MDCSimpleInkLayer.m

```diff
@@ -0,0 +1,162 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCSimpleInkLayer.h"
+
+@implementation MDCSimpleInkLayer
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    [self commonMDCSimpleInkLayerInit];
+  }
+  return self;
+}
+
+- (instancetype)initWithLayer:(id)layer {
+  self = [super initWithLayer:layer];
+  if (self) {
+    [self commonMDCSimpleInkLayerInit];
+  }
+  return self;
+}
+
+- (void)commonMDCSimpleInkLayerInit {
```

Comment:
Since init is the only designated initializer, move the body content to init and delete this method.

Reply:
Done.